spec/views/forms/show.html.erb_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+describe "forms/show.html.erb" do
+  around do |example|
+    ClimateControl.modify RUNNER_BASE: "api-host" do
```

Comment:
I guess api-host might be better as runner-host in this case? not suggesting changing it. Would it be worth adding a helper or config set by the env var to keep setting the value in one place in teh code somewhere? Don't know if the benefits outway the cost, just a thought

Reply:
🤦  good catch, should have been `runner-host` not `api-host`